.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -6,58 +6,48 @@
  * 1) 
  */
 .global-header {
+  position: relative;
   margin-bottom: var(--eds-size-4);
 
+  &:hover,
+  &:focus-within {
+    @media all and (min-width: $eds-bp-lg) {
```

Comment:
Just curious – why are we using `all`? Isn't that the default? Do we need to specify it?